dev/.env.docker-compose-native

```
@@ -10,3 +10,4 @@ AWS_REQUEST_PAYER=requester
 RDWATCH_POSTGRESQL_URI="postgresql://rdwatch:secretkey@localhost:5432/rdwatch"
 RDWATCH_REDIS_URI="redis://:secretkey@localhost:6379"
 RDWATCH_CELERY_BROKER_URL="amqp://rdwatch:secretkey@localhost:5672/"
+RDWATCH_POSTGRESQL_SCORING_URI="postgresql://scoring:secretkey@scoredb:5432/scoring"
```

One more issue i just noticed when testing this with native django
```suggestion
RDWATCH_POSTGRESQL_SCORING_URI="postgresql://scoring:secretkey@localhost:5433/scoring"
```